examples/with-typescript-graphql/.graphql-let.yml

```diff
@@ -1,7 +1,7 @@
-schema: lib/type-defs.graphqls
+schema: '**/*.graphqls'
+schemaEntrypoint: 'lib/type-defs.graphqls'
 documents: '**/*.graphql'
 plugins:
   - typescript
   - typescript-operations
   - typescript-react-apollo
-respectGitIgnore: true
```

Comment:
Why is this not required anymore?

Reply:
Thanks for asking. That is because now it becomes [the default value](https://github.com/piglovesyou/graphql-let#graphql-let-specific-options).